e2e/windows/nodebootstrapping_template.json

```diff
@@ -8,11 +8,11 @@
         "properties": {
             "orchestratorProfile": {
                 "orchestratorType": "Kubernetes",
-                "orchestratorVersion": "1.24.0",
```

Comment:
I still think the change in nodebootstrapping_template.json is unnecessary. You can remove these code change and test again.

Reply:
Removed and it worked.